dist/jquery.mark.es6.js

```diff
@@ -70,7 +70,7 @@
                 return;
             }
             if (typeof log === "object" && typeof log[level] === "function") {
-                log[level](`mark.js: ${msg}`);
+                log[level](`mark.js: ${ msg }`);
```

Comment:
Ok, I ran it too, so I really have no idea. We both must have different versions of npm dependencies, but there's no reason why.